index.html

```diff
@@ -65,15 +65,194 @@ <h1 class="intro-heading">
 							</a>
 						</li>
 					</ul>
-				</section>
-				<section class="action">
-					<a href="#" class="action-button"
-						><img
+				</div>
+				<div class="action">
+					<a href="#" class="action-button">
+						<img
 							class="action-button__active"
 							src="/images/Icon-down-button.png"
-							alt=""
-					/></a>
-				</section>
+							alt="" />
+					</a>
+				</div>
+			</section>
+			<section class="works">
+				<div class="header-container">
+					<h2 class="works-header">My Recent Works</h2>
+					<div class="indicator-container">
+						<img src="/images/Indicator.png" alt="" />
+					</div>
+				</div>
+				<div class="cards-work">
+					<div class="card">
+						<img src="/images/Img Plaholder.png" alt="" />
+						<div class="title-container">
+							<h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+						</div>
+						<ul class="tech">
+							<li>Ruby on Rails</li>
+							<li>CSS</li>
+							<li>Javascript</li>
+							<li>HTML</li>
+						</ul>
+						<div class="button-container">
+							<button type="button" class="project-button">See Project</button>
+						</div>
+					</div>
+					<div class="card">
+						<img src="/images/Img Plaholder.png" alt="" />
```

Comment:
- Kindly reduce the size of the card I can just suggest you instead of using the image you can just use the border-box and give it the grey background or you can also simply resize the image :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/227666372-dc341ed9-04b9-4eb1-9005-84e4b70829bc.png)

## Figma design

![image](https://user-images.githubusercontent.com/95297251/227666570-d726f818-528e-471d-aec4-90cb559b8658.png)